etcdctl/ctlv2/command/util.go

```diff
@@ -78,7 +79,7 @@ func getPeersFlagValue(c *cli.Context) []string {
 
 	// If we still don't have peers, use a default
 	if peerstr == "" {
-		peerstr = "http://127.0.0.1:2379,http://127.0.0.1:4001"
```

Comment:
@heyitsanthony i was not talking about the initial log but the logs which follow after each request.
`2016-10-06 03:26:01.623455 D | etcdserver/api/v2http: [GET] /v2/members remote:127.0.0.1:50627
2016-10-06 03:26:01.705381 D | etcdserver/api/v2http: [GET] /v2/members/leader remote:127.0.0.1:50627
2016-10-06 03:26:16.559901 D | etcdserver/api/v2http: [PUT] /v2/keys/foo remote:127.0.0.1:50632
2016-10-06 03:26:25.157198 D | etcdserver/api/v2http: [GET] /v2/keys/foo?quorum=false&recursive=false&sorted=false remot`